### ISSUE TITLE
Decryption of Nexus 4 Fails

### ISSUE THREAD

**Issue by soziophobic** [TeamWin/Team-Win-Recovery-Project#209]
Hi,

I have a stock Nexus 4 (rooted and encrypted) 

After booting into recovery via:

fastboot -c "lge.kcal=0|0|0|x" boot openrecovery-twrp-2.6.2.0-mako.img

I get a "Please Enter Your Password" prompt.
After entering the correct password I get "Password Failed. Please Try Again"

I get this same behavior in all TWRP recoveries from 2.4.4.0 to 2.6.2.0

recovery.log: http://pastebin.com/ScGjHuqz

**Comment by davidjirovec** [comment]
Hi, did you have any success solving this issue?

**Comment by soziophobic** [comment]
Hi, no I tried a few versions after this but none worked, and gave up.

**Comment by Dees-Troy** [comment]
Data decrypt works fine on my Nexus 4. Unable to reproduce.

**Comment by soziophobic** [comment]
Hi, I tried it again. And it doesn't work for me.

Steps I did:
1. encrypt phone (version 4.4.2. build: KOT49H)
2. change password with "cryptfs password" from the play store
3. boot to recovery with: fastboot -c "lge.kcal=0|0|0|x" boot /mypath/openrecovery-twrp-2.6.3.3-mako.img
4. enter correct password. Error message as in 1 post. Same error in recovery log

Didn't think about testing it before changing the password. Maybe that is the Problem.
